src/components/Auth/LoginForm.js

```diff
@@ -0,0 +1,118 @@
+import React, { useState, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Link, Redirect } from 'react-router-dom';
+import { Box, Button, TextField } from '@material-ui/core/';
+import { useAuth } from './AuthContext';
+import axios from 'axios';
+
+const LoginForm = ({ toggleActiveForm }) => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+  const auth = useAuth();
+  const referer = '/profile';
+
+  const handleLogin = e => {
+    e.preventDefault();
+    const data = {
+      username,
+      password,
+    };
+    axios
+      .post('/auth/obtain_token/', data)
+      .then(res => {
+        auth.setAuthTokens(res.data);
+        setIsLoggedIn(true);
+      })
+      .catch(error => {
+        if (error.response) {
+          setErrorMessage(error.response.data);
+        } else {
+          setErrorMessage('There was an error!');
+        }
+      });
+  };
+
+  if (isLoggedIn) {
+    return <Redirect to={referer} />;
+  }
+  return (
+    <Fragment>
```

Comment:
A bit of personal preference, but would prefer to shorthand `<Fragment>` to `<>` (I think it keeps the JSX cleaner with fewer things to read)

Reply:
Ahh, the reason I'm used to `<Fragment>` instead of the latter is because at work, the linter actually complains if we use `<>` instead of `<Fragment>` 😅 For personal projects that are more up to date with the latest React version I do like the latter, too. Will update this.

Reply:
(Done in 1b1f4c9)